ui/components/ReactSelectWrapper.js

```diff
@@ -123,8 +121,18 @@ function MultiValue(props) {
 }
 
 function Menu(props) {
+  const theme = useTheme();
   return (
-    <Paper square className={props.selectProps.classes.paper} {...props.innerProps}>
+    <Paper
+      square
+      {...props.innerProps}
+      sx={{
+        zIndex: 9999,
```

Comment:
Why so high? How does this align with the Notification Center, modals, toast notifications?